mac

```diff
@@ -22,12 +22,12 @@ mysql_password="root"
 mysqldump="/Applications/MAMP/Library/bin/mysqldump"
 dropbox_folder="/Volumes/Cloud/Dropbox/"
 php="/Applications/MAMP/bin/php/php5.5.26/bin/php"
-php_cli="/usr/local/bin/php"
+php_cli=$(which php)
 phpmyadmin="http://localhost/phpMyAdmin/"
-github_email_account="guarinogabriel@gmail.com"
-github_username="guarinogabriel"
+github_email_account=$(git config -l | grep user.email=.* | cut -d "=" -f2)
+github_username=$(git config -l | grep user.name=.* | cut -d "=" -f2)
```

Comment:
both commands work fine for me (same for email). Just wondering whats causing .* one to break for you?